csrankings.csv

```diff
@@ -2046,6 +2046,7 @@ Byron Boots,Georgia Institute of Technology,http://www.cc.gatech.edu/~bboots3,kX
 Byron C. Wallace,Northeastern University,http://www.byronwallace.com,KTzRHmwAAAAJ
 Byron Choi,Hong Kong Baptist University,http://www.comp.hkbu.edu.hk/~bchoi,S6L1eOMAAAAJ
 Byron J. Williams,Mississippi State University,http://www.fsnhp.msstate.edu/docs/staff/Byron%20%20Williams%20CV%20July%202012%20.pdf,t4bGLWUAAAAJ
+Byron Yu,Carnegie Mellon University,http://users.ece.cmu.edu/~byronyu/,Fz3_tukAAAAJ

```

Comment:
Not here: https://www.cs.cmu.edu/directory

Reply:
SCS directory is not complete or frequently updated to be current. Different departments maintain their own frequently updated personnel lists. See Byron's affiliation [here](https://www.ml.cmu.edu/people/affiliated-faculty.html) Also see his advisees [here](https://www.ml.cmu.edu/people/phd-students.html)